week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -39,6 +59,13 @@ Write a function that:
 
 function formatPercentage(arr) {
```

Comment:
You got most of this correct and most inputs will output the right results (after fixing the forgotten right brace `}`). Just a few minor issues that are easy to miss that can make the solution output the correct values for all inputs as expected.